Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Components/Objects/SpeckleBlockDefinitionPassthrough.cs

```diff
@@ -110,18 +109,38 @@ protected override void SolveInstance(IGH_DataAccess da)
 
       foreach (var objGoo in inputObjects)
       {
-        if (objGoo?.Value != null)
+        SpeckleObjectWrapper? obj = null;
+
+        // Handle SpeckleObjectWrapper
+        if (objGoo is SpeckleObjectWrapperGoo speckleObjGoo && speckleObjGoo.Value != null)
```

Comment:
This case, and the generic case below, can be captured in the same statement. 

I'd probably do a logic flow like this:
> create new objectGoo and try cast (handles object wrapper, model objects, and loose geo)
> if failed, create new instanceGoo and try cast (handles instancegoo, 
> if both failed, add a warning that object of type <> could not be added to definition